test/cluster/conftest.py

```diff
@@ -55,12 +56,7 @@
 def pytest_addoption(parser):
     parser.addoption('--manager-api', action='store',
                      help='Manager unix socket path')
-    parser.addoption('--host', action='store', default='localhost',
-                     help='CQL server host to connect to')
-    parser.addoption('--port', action='store', default='9042',
-                     help='CQL server port to connect to')
-    parser.addoption('--ssl', action='store_true',
-                     help='Connect to CQL via an encrypted TLSv1.2 connection')
+    add_cql_connection_options(parser)
```

Comment:
Looking at the implementation below, it seems to me that add_cql_connection_options() only adds port and ssl, but not "host", and you also need to call add_host_option(). What am I missing?